### ISSUE TITLE
Identify links in the media contact field using something besides color.

### ISSUE THREAD

**Issue by imcbride** [middlebury/frontend#137]
Or increase the color contrast ratio to 3:1. These links are added by the editor, so we can't automatically apply a class to them in the template. They should be underlined, bold, or something else to satisfy WCAG 1.4.1.

![screen shot 2018-03-12 at 2 09 51 pm](https://user-images.githubusercontent.com/662895/37301483-0f80dc5c-25ff-11e8-8f81-d34a998b8bcf.png)

**Comment by imcbride** [comment]
I'm re-opening this to add a few more instances. First up: links in event teaser blurbs. https://www.middlebury.edu/institute/events?page=42

![screen shot 2018-03-20 at 9 49 40 am](https://user-images.githubusercontent.com/662895/37658571-3c1d6200-2c24-11e8-9bca-f1dcf45a3e18.png)

**Comment by imcbride** [comment]
Related stories teaser blurbs. https://www.middlebury.edu/institute/news/ipss-spotlight-aricquel-payne-six-square

![screen shot 2018-03-20 at 9 52 12 am](https://user-images.githubusercontent.com/662895/37658660-6fb199d8-2c24-11e8-9028-630a09961803.png)

**Comment by imcbride** [comment]
And image captions. https://www.middlebury.edu/institute/news/anonymous-donor-pledges-18-million-support-center-blue-economy

![screen shot 2018-03-20 at 9 53 16 am](https://user-images.githubusercontent.com/662895/37658713-8ea0e66e-2c24-11e8-9a53-0aa93e83c0d7.png)
